geode-docs/managing/security/implementing_authorization.html.md.erb

```diff
@@ -276,3 +276,16 @@ for details about this property.
 - Define any extra resources that the implemented authorization algorithm
 needs in order to make a decision.
 
+### <a id="AuthorizeFcnExecution" class="no-quick-link"></a>Authorization of Function Execution
+
+By default, a function executed on servers requires that the
+entity invoking the function have `DATA:WRITE` permission on the
+region(s) involved.
+Since the default permission may not be appropriate for all functions,
+the permissions required may be altered.
+
+To implement a different set of permissions,
+implement an overloaded `Function.getRequiredPermissions()` method
```

Comment:
I think it might be more correct to say 
```
implement the `Function.getRequiredPermissions()` method
``` 
or
```
override the `Function.getRequiredPermissions()` method
```
(**Overload** has a slightly different meaning in Java than is being used here.)  Also, since we are instructing the user in what they _ought_ to do, should this read 
```
The method should return a Collection of the permissions required
```

rather than 
```
The method returns a Collection of the permissions required
```